helpt/urls.py

```diff
@@ -17,6 +17,7 @@
 from django.contrib import admin
 from rest_framework.routers import DefaultRouter
 
+from projects.adapters.github_hook import urls as github_urls
```

Comment:
Hmm, maybe some kind of adapter registry would work better here. Let's go with your original solution for now.